xm-commons-topic/src/main/java/com/icthh/xm/commons/topic/message/MessageService.java

```diff
@@ -0,0 +1,20 @@
+package com.icthh.xm.commons.topic.message;
+
+import com.icthh.xm.commons.lep.LogicExtensionPoint;
+import com.icthh.xm.commons.lep.spring.LepService;
+import com.icthh.xm.commons.topic.domain.TopicConfig;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Slf4j
+@Service
+@Transactional
+@LepService(group = "topic")
+public class MessageService {
+
+    @LogicExtensionPoint(value = "TopicMessage", resolver = MessageTypeKeyResolver.class)
```

Comment:
suggest to rename `value = "TopicMessage"` to `value = "OnMessage"`

Reply:
done